media-gfx/nifskope/nifskope-2.0.dev9.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="Tool for opening and editing NIF files"
+DESCRIPTION="NifSkope is a tool for opening and editing the NetImmerse file format (NIF). \
+NIF is used by video games such as Morrowind, Oblivion, Skyrim, Fallout 3/NV/4/76, Starfield, \
+Civilization IV, and more."
+HOMEPAGE="http://www.niftools.org/"
+COPYRIGHT="2005-2014 NIF File Format Library and Tools"
+LICENSE="NIFSKOPE"
+REVISION="1"
+SOURCE_URI="https://github.com/hexabits/nifskope/archive/refs/tags/v2.0.dev9.tar.gz"
+CHECKSUM_SHA256="b40baca5fc3b11292cd284b5ef0d07dbe40c3bab8067c05b7f25bdb6141fe26c"
+SOURCE_URI_2="https://github.com/g-truc/gli/archive/refs/tags/0.8.2.0.tar.gz"
+CHECKSUM_SHA256_2="9e7024c2df77c011eff4f66667c1834620c70b7902cd50f32ab48edd49fe0139"
+SOURCE_URI_3="https://github.com/qhull/qhull/archive/refs/tags/v8.1-alpha1.tar.gz"
+CHECKSUM_SHA256_3="09e5e4c5b2b8a9e617a46876fef5a3d33e70aa1d08a163ff05d37701327c3be7"
+SOURCE_URI_4="https://github.com/niftools/nifxml/archive/refs/tags/v0.9.0.tar.gz"
+CHECKSUM_SHA256_4="984c247115bc49f90ded69b4c8feb9a62a0365ad53f5890312302b6a05bb4ed7"
+SOURCE_URI_5="https://raw.githubusercontent.com/niftools/kfmxml/develop/kfm.xml#noarchive"
+CHECKSUM_SHA256_5="3af5634dfef643494bb7646924b1b18ab22bc0d9b5ea4758d4e6b693c78a2f3e"
+
+PATCHES="nifskope-2.0.dev9.patchset"
+ADDITIONAL_FILES="nifskope.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/nifskope/kfm.xml keep-old
+	settings/nifskope/nif.xml keep-old	
+	"
+	
+PROVIDES="
+	nifskope$secondaryArchSuffix = $portVersion
+	app:NifSkope$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5OpenGL$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix    
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:glm
+	devel:libqhull_r$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	cmd:clang++ >= 18
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix
+	cmd:which
+	cmd:wget
```

Comment:
Sort alphabeticly, do you still need wget?

Reply:
Indeed wget is no more needed. Fixed.